mods/species/utility_frames/species_bodytypes.dm

```diff
@@ -4,6 +4,10 @@
 	bodytype_category = BODYTYPE_HUMANOID
 	limb_blend =        ICON_MULTIPLY
 
+	bodyfall_sounds = list(
```

Comment:
Would suggest moving the clonk sound onto the base decl as `synthetic_bodyfall_sounds` and checking `isSynthetic()` in `UpdateLyingBuckledAndVerbStatus()` to decide which list to pick from.